src/test/java/org/opentripplanner/updater/trip/TimetableSnapshotSourceTest.java

```diff
@@ -1052,14 +1167,97 @@ public void repeatedlyAddedTripWithNewRoute() {
         List.of(tripUpdate),
         feedId
       );
-      var secondPattern = assertAddedTrip(SERVICE_DATE, addedTripId, updater);
+      var secondPattern = assertAddedTrip(SERVICE_DATE, addedTripId, updater, false);
       var secondRoute = secondPattern.getRoute();
 
       // THEN
 
       assertSame(firstRoute, secondRoute);
       assertNotNull(transitModel.getTransitModelIndex().getRouteForId(firstRoute.getId()));
     }
+
+    static List<Arguments> addedRemovalTestCase() {
```

Comment:
Should this parameter-generation method's name be changed to exactly match the test it's parameterizing? For example: `cancelingAddedTripTestCases()`;